book/src/pod-info.md

```diff
@@ -17,21 +17,18 @@ Specifically the `podInfoOnMountVersion` field instructs Kubernetes that the CSI
 For example, the existence of the following object would cause Kubernetes to add pod information at mount time to the `NodePublishVolumeRequest.volume_context` map.
 
 ```
-apiVersion: csi.storage.k8s.io/v1alpha1
+apiVersion: storage.k8s.io/v1beta1
 kind: CSIDriver
 metadata:
   name: testcsidriver.example.com
 spec:
-  podInfoOnMountVersion: v1
+  podInfoOnMountVersion: true
 ```
 
-There is only one `podInfoOnMountVersion` version currently supported: `v1`.
-
-The value `v1` for `podInfoOnMountVersion` will result in the following key/values being added to `publish_context`:
+`true` value in `podInfoOnMountVersion` will result in the following key/values being added to `publish_context`:
```

Comment:
If the `podInfoOnMount` field is set to `true`, during mount, Kubelet will add the following key/values to the `publish_context` field in the CSI `NodePublishVolumeRequest`: